openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -14,9 +14,21 @@
       "description": "Name of a table. Expected to be unique within a database.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 128,
+      "maxLength": 512,
       "pattern": "^((?!::).)*$"
     },
+    "fullyQualifiedEntityName": {
+      "description": "A unique name that identifies a table. Example for table 'DatabaseService.Database.Schema.Table'.",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 896
```

Comment:
we should refer to common fullyQualifiedName in basic.json

Reply:
as discussed, using the common FQN and removing the max limit size in there